recipe/activate.sh

```diff
@@ -27,6 +27,7 @@ export CFLAGS="${CFLAGS} ${CUDA_CFLAGS} ${CUDA_LDFLAGS}"
 export CPPFLAGS="${CPPFLAGS} ${CUDA_CFLAGS} ${CUDA_LDFLAGS}"
 export CXXFLAGS="${CXXFLAGS} ${CUDA_CFLAGS} ${CUDA_LDFLAGS}"
 export LDFLAGS="${LDFLAGS} ${CUDA_LDFLAGS}"
+export CUDAFLAGS="${CUDAFLAGS} ${CUDA_CFLAGS}"
```

Comment:
Do we want the `LDFLAGS` too or just `CFLAGS`?

```suggestion
export CUDAFLAGS="${CUDAFLAGS} ${CUDA_CFLAGS} ${CUDA_LDFLAGS}"
```

Reply:
I don't think the library paths are needed because NVCC isn't being used as a linker? For example, the libmagma feedstock (which has separable compilation off for linux) uses GCC to link all of the generated object files as the last step, not NVCC. The library paths are only needed at link-time, the headers are only needed at compile time?

Reply:
I'm not sure if people ever use the NVCC CLI for linking.

Reply:
`CUDA_LDFLAGS` should only be in LDFLAGS. I don't know why `CUDA_LDFLAGS` was added to `CXXFLAGS` and friends. That's clearly wrong.

Reply:
Maybe this should go in a separate issue or PR? Doesn't seem related to Daniel's initial request